app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -57,19 +57,24 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
 
 If you are interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for a single payment of £10,000 from the UK government. [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment). 
 
-## Apply for qualified teacher status (QTS)
+## English qualified teacher status (QTS)
```

Comment:
Could people get confused if they speak English? I don't know if Qualified teacher status (QTS) in England is clearer?

Reply:
We decided to qualify QTS with 'English' because users who are already qualified teachers assumed they already had QTS. ie didn't realise it indicated a specific status in the English teaching profession...